### PR TITLE
Change cookie expiration time from 14 days to 6 hours

### PR DESCRIPTION
**Pull request by skofman1**
@blowdart @maartenba

### CODE REVIEW

File: src/NuGetGallery/Authentication/Providers/LocalUser/LocalUserAuthenticator.cs [hunk 1]
```diff
@@ -23,7 +24,8 @@ protected override void AttachToOwinApp(IGalleryConfigurationService config, IAp
                 AuthenticationMode = AuthenticationMode.Active,
                 CookieHttpOnly = true,
                 CookieSecure = cookieSecurity,
-                LoginPath = new PathString("/users/account/LogOn")
+                LoginPath = new PathString("/users/account/LogOn"),
+                ExpireTimeSpan = TimeSpan.FromHours(6)
```
Comment (scottbommarito) [inline src/NuGetGallery/Authentication/Providers/LocalUser/LocalUserAuthenticator.cs]: So we didn't specify the expiration before, and the default is 14 days?

**Comment by blowdart** [comment]
Might want to ensure sliding expiration is true, so if people are using the side for 6 hours straight, their cookie will get refreshed for another six;

https://msdn.microsoft.com/en-us/library/microsoft.owin.security.cookies.cookieauthenticationoptions.slidingexpiration%28v=vs.113%29.aspx?f=255&MSPPError=-2147217396

File: src/NuGetGallery/Authentication/Providers/LocalUser/LocalUserAuthenticator.cs (hunk 1 above)
Comment (skofman1) [inline src/NuGetGallery/Authentication/Providers/LocalUser/LocalUserAuthenticator.cs]: yup

**Comment by skofman1** [comment]
This is the default, but I overridden it in code just to make sure :)

**Pull request merged**